lua/wire/gates/string.lua

```diff
@@ -178,6 +178,12 @@ GateActions["string_replace"] = {
 	inputtypes = { "STRING" , "STRING" , "STRING" },
 	outputtypes = { "STRING" },
 	output = function(gate, A, B, C)
+		if  (A and #A or 0)
```

Comment:
Could you move this after the lines 187-189 so you don't need the 'and' 'or' checks?

Reply:
good call